### ISSUE TITLE
1.13 support.

### ISSUE THREAD

**Issue by TheIntelloBox** [IntellectualSites/PlotSquared#2025]
https://www.spigotmc.org/threads/bukkit-craftbukkit-spigot-bungeecord-1-13-pre7-development-builds.328883/

**Comment by RedstoneFuture** [comment]
Yes, we would have forgotten otherwise. :D :P

**Comment by Speentie8081** [comment]
I've been waiting for 3 days, literally no progress on adding 1.13 support to PlotSquared ...

**Comment by Chazotu** [comment]
Spigot Released the 1.13 Release De Build to day so maby their will be an update around here in future time

**Comment by RedstoneFuture** [comment]
Yes. But 1.13 was a big update with many chances. This need time.

**Comment by Speentie8081** [comment]
@RedstoneFuture I already submitted a pull request that adds complete 1.13 support to plotsquared haha
